src/mgmt/cli_mgmt.c

```diff
@@ -300,6 +302,7 @@ get_attach_info(const char *name, int *npsrs, struct dc_mgmt_psr **psrs)
 
 	/* Prepare the GetAttachInfo request. */
 	req.sys = (char *)name;
+	req.numa = 1; // to be replaced by this client's numa node
```

Comment:
(style)  do not use C99 // comments

Reply:
This assignment is just here for debug. It will be given the actual NUMA node as part of DAOS-4348.